src/systems/hud/hud.frag

```diff
@@ -0,0 +1,36 @@
+precision mediump float;
+
+uniform sampler2D u_spritesheet;
+uniform vec4 u_stencils[5];
+uniform vec4 u_sprites[5];
```

Comment:
Should the sprite coordinates array be a vector of integers instead of a vector of floats?

(It seems the stencil code could also be rewritten to represent the stencil coordinates as integers instead of floats, if we wanted.)